tutorials/avail-da.md

```diff
@@ -6,7 +6,52 @@ Avail DA offers scalable data availability that underpins the Avail ecosystem an
 
 This tutorial serves as a comprehensive guide for deploying your GM world rollup on Avail's data availability (DA) network.
 
-Before proceeding, ensure that you have completed the [GM world rollup](/tutorials/gm-world) tutorial, which covers setting up a local sovereign gm-world rollup and connecting it to a local (mock) DA node.
+Before proceeding, ensure that you have completed the GM World rollup setup. Follow the steps below to install Rollkit and GM World.
+
+## 🧑‍🔧 Install Rollkit (CLI)
+You should have the Rollkit CLI and Golang installed already. 
+
+If not, to install Rollkit, run the following command in your terminal:
+```bash
+curl -sSL https://rollkit.dev/install.sh | sh -s v0.13.4
```

Comment:
Please use the constant `rollkitLatestTag ` here, refer to https://github.com/rollkit/docs/blob/321d862e5665f430e93e78eb7dabc5a6f69c4b66/tutorials/quick-start.md?plain=1#L18